serverlessworkflow/modules/ROOT/nav.adoc

```diff
@@ -21,7 +21,7 @@
 ** xref:service-orchestration/orchestration-of-grpc-services.adoc[Orchestration of gRPC based services]
 //* Eventing
```

Comment:
```suggestion
* Eventing
```

Reply:
This JIRA depends on the callback one, which already has this change